.buildkite/integration.pipeline.yml

```diff
@@ -8,14 +8,16 @@ steps:
   - group: "Integration tests: packaging"
     key: "int-packaging"
     steps:
-      - label: "Packaging: Ubuntu x86_64"
-        key: "packaging-ubuntu-x86-64"
+      # Build matrix is not used for packaging in favor to unique step keys
+      # Packaging linux/amd64
+      - label: "Packaging: linux/amd64 rpm"
```

Comment:
should we update this to show rpm,deb,tgz since it's doing all three?

Reply:
> It is still possible to use the build matrix with 2 axes for arm and x86. The comment is added for questions like [this one](https://github.com/elastic/elastic-agent/pull/6971/files/249a6f0c1e82f61029a4a8a54b150fad059a8ecf#r1967170557)

Just to clarify the ask here, I am just proposing the following change

FROM:
```
      - label: "Packaging: linux/amd64 rpm"
```

TO

```
      - label: "Packaging: linux/amd64 tag.gz,rpm,deb"
```

It's ok, though, that this label gets fixed in a follow up PR